src/App.js

```diff
@@ -26,54 +28,76 @@ class App extends Component {
         currentUser: user,
       });
     }
+    
   }
 
   logOut() {
     AuthService.logout();
   }
 
   render() {
-    const { currentUser} = this.state;
+    
+    const { currentUser } = this.state;
 
     return (
       <div>
         <nav className="navbar navbar-expand navbar-dark bg-dark">
-          <div className="navbar-nav mr-auto">
-          </div>
 
-          {currentUser ? (
-            <div className="navbar-nav ml-auto">
-              <li className="nav-item">
-                <Link to={"/profile"} className="nav-link">
-                  {currentUser.username}
-                </Link>
-              </li>
-              <li className="nav-item">
-                <a href="/login" className="nav-link" onClick={this.logOut}>
-                  LogOut
-                </a>
-              </li>
-            </div>
+          {currentUser != undefined ?  (
```

Comment:
Javascript has the `!==` operator as well as the `!=` operator; here, the difference is that `!==` won't cause type coercion, and should be used instead, but I don't think it will cause problems here. I'm just bringing it up in case you didn't know about it.